concepts/05 UI Components/DataGrid/00 Getting Started with DataGrid/80 Export Data.md

```diff
@@ -0,0 +1,289 @@
+DataGrid allows users to export grid data as Excel and PDF documents. This feature requires the following third-party libraries:
+
+- Excel: <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> v4+ and <a href="https://github.com/eligrey/FileSaver.js/" target="_blank">FileSaver</a> v2.0.2+
+
+- PDF: <a href="https://github.com/parallax/jsPDF" target="_blank">jsPDF</a>
+
+To configure export operations, use the **excelExporter**.[exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/excelExporter/#exportDataGridoptions) or **pdfExporter**.[exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/pdfExporter/#exportDataGridoptions) method. It requires a DataGrid instance and a target Excel worksheet or a PDF document.
```

Comment:
You have this paragraph about `exportDataGrid` methods, but then the first sentence in the next paragraphs continues to talk about it. It even refers to it as "this method". I would rather have it all in one paragraph and then you can start the next paragraph with a sentence about the example. 

```suggestion
To configure export operations, use the **excelExporter**.[exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/excelExporter/#exportDataGridoptions) or **pdfExporter**.[exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/pdfExporter/#exportDataGridoptions) method. It requires a DataGrid instance and a target Excel worksheet or a PDF document. You can call these methods at any point in your application.
```